getit/getit.py

```diff
@@ -198,7 +200,12 @@ def main():
     system = ''
     fname = ''
     info = ''
-    path = "C:\\Users\\" + getuser() + "\\Downloads\\getit\\"
+    if (platform.system() == "Windows"):
+        path = "C:\\Users\\" + getuser() + "\\Downloads\\getit\\"
+    else:
+        path = "/Users/" + getuser() + "/Downloads/getit"
```

Comment:
It should not be `Users` in path for Linux, It should be `home` and check for the other platforms also and pls try to run it on your own system first.

Reply:
Uhh, sorry for that. I thought it would be the same on linux as macOS.